src/flow_solver/flow_solver_cases/periodic_cube_flow.cpp

```diff
@@ -77,6 +77,20 @@ void PeriodicCubeFlow<dim,nstate>::display_grid_parameters() const
     if constexpr(dim==1) this->pcout << "- - Domain length: " << this->domain_size << std::endl;
     if constexpr(dim==2) this->pcout << "- - Domain area: " << this->domain_size << std::endl;
     if constexpr(dim==3) this->pcout << "- - Domain volume: " << this->domain_size << std::endl;
+
+    std::string c_parameter_string;
+    using FREnum = Parameters::AllParameters::Flux_Reconstruction;
+    FREnum fr_type = this->all_param.flux_reconstruction_type;
+    if (fr_type == FREnum::cDG)              c_parameter_string = "cDG";
+    else if (fr_type == FREnum::cSD)         c_parameter_string = "cSD";
+    else if (fr_type == FREnum::cHU)         c_parameter_string = "cHU";
+    else if (fr_type == FREnum::cNegative)   c_parameter_string = "cNegative";
+    else if (fr_type == FREnum::cNegative2)  c_parameter_string = "cNegative2";
+    else if (fr_type == FREnum::cPlus)       c_parameter_string = "cPlus";
+    else if (fr_type == FREnum::c10Thousand) c_parameter_string = "c10Thousand";
+    else if (fr_type == FREnum::cHULumped)   c_parameter_string = "cHULumped";
+
+    this->pcout << "- Using c parameter: " << c_parameter_string << std::endl;
```

Comment:
maybe move this to `FlowSolverCaseBase<dim,nstate>::display_flow_solver_setup` and only print this if using strong form + flux reconstruction

Reply:
Sure, that makes sense. I'll only check `use_weak_form`, since printing cDG will tell the user that FR isn't being used. I'll also add a line that prints whether strong or weak is being used since that's already checked.